### ISSUE TITLE
Cannot make new image occlusion card

### ISSUE THREAD

**Issue by Jeffx9** [glutanimate/image-occlusion-enhanced#144]
## Problem description

- Followed official video and various guides step by step to make image occlusion card

- General overview = installed image occlusion enhancer 2.1 alpha via code provided on Anki website --> went to basic card creation --> I clicked the add-on to insert image in the top right hand corner --> I selected the image --> I masked using a rectangle some of the information and clicked "hide all, guess one". This inserted the cards --> When I try and do the cards, the front of the card shows up as the hidden (ID) code for example "bd9e6e32110a4a1487b573e86282032d-ao-2". The back of the card shows up as the answer.

- I have tried using both PC and MAC with the same issue
- I have tried uninstalling/reinstalling with no success
- I have tried using different image types with no success
- I have tried to sync/restart the app with no success
- I have tried to fill in some of the extra bits of information on the card such as header/footer with no success
- I have tried changing the card type to cloze with no success

Any ideas?

#### Checklist

*Please replace the space inside the brackets with an **x** if the following items apply:*

 - [x] I've restarted Anki to see if it helps
 - [x] I've verified that I use the latest version of the add-on by redownloading it from AnkiWeb
 - [x] I've verified that I use the latest version of Anki by checking at https://apps.ankiweb.net#download
 - [x] I've tried to disable other add-ons to see if there are any interactions present
 - [ ] My issue disappears when I hold shift while starting Anki.
 - [x] I've checked if anyone else reported this problem before by looking through the issue reports. I also checked to see if there is a section about known issues in the add-on description, documentation, or README.
 

#### Information about your Anki set-up

*Please fill out the section corresponding with your Anki version:*

##### If you are using Anki 2.1

*Please open Anki, go to Help → About, click on "Copy Debug Info", and paste the result between the backticks below (if the button does not appear you are using an older version of Anki 2.1 and will need to update first):*
```
Anki 2.1.15 (442df9d6) Python 3.6.7 Qt 5.12.1 PyQt 5.11.3
Platform: Mac 10.15.3
Flags: frz=True ao=True sv=1
```
Add-ons:

Image Occlusion Enhanced for Anki 21 alpha

#####  If you are using Anki 2.0

*Please open Anki, go to Help → About, find the two lines starting with `Version:` and `Qt`, and copy-paste them between the backticks below:*

```

```

*Please fill in details about your operating system (Windows/macOS/Linux, which version):*

OS = MacOS Catalina
  Model Name:	MacBook Pro
  Model Identifier:	MacBookPro11,3
  Processor Name:	Quad-Core Intel Core i7
  Processor Speed:	2.5 GHz
  Number of Processors:	1
  Total Number of Cores:	4
  L2 Cache (per Core):	256 KB
  L3 Cache:	6 MB
  Hyper-Threading Technology:	Enabled
  Memory:	16 GB

NOTE: Also tried using Windows/PC and the same error occurs!

*Please open Anki, go to Tools → Add-ons, take a screenshot of your installed add-ons, and paste it below:*

<img width="1259" alt="Screen Shot 2020-04-13 at 11 18 04 pm" src="https://user-images.githubusercontent.com/63594168/79123230-07c7cb00-7ddd-11ea-958f-918e31162d06.png">

#### Error message (if any)

*If you've received an error message, please copy and paste it between the backticks below:*

<img width="1344" alt="Screen Shot 2020-04-13 at 11 20 48 pm" src="https://user-images.githubusercontent.com/63594168/79123396-6ab96200-7ddd-11ea-8ea6-b9deb97a178b.png">

<img width="1341" alt="Screen Shot 2020-04-13 at 11 21 45 pm" src="https://user-images.githubusercontent.com/63594168/79123452-8d4b7b00-7ddd-11ea-8c74-236efc0b64a5.png">

**Comment by glutanimate** [comment]
Hi Jeff,

Thanks for the detailed bug report. It looks like your image occlusion card template got corrupted somehow.

Please head to Tools → Manage Note Types and rename the "Image Occlusion Enhanced" note type to something random. Then restart Anki and try to create some new image occlusion cards. This will prompt the add-on to generate a fresh note type with the defaults.

To fix your old notes, you could either try [resetting the renamed note type's card template to the defaults](https://github.com/glutanimate/image-occlusion-enhanced/wiki/Troubleshooting#card-template), or change the note type of the old notes to the newly created "Image Occlusion Enhanced" note type. You can do this in Anki's card browser: https://docs.ankiweb.net/#/browsing?id=other-menu-items

To prevent the card templates from being corrupted in the future, it would be important to know what caused the corruption in the first place. Safe for another add-on messing with things or importing a corrupted template from an outside deck, this is usually caused by user interaction. Do you remember tinkering with the templates in any way? E.g. adding some tweaks you found online or something like that?

In the future I'm likely going to add an in-app option to restore the card templates to the defaults, as it seems like this is happening to quite a few users. It might also be worthwhile exploring to lock down the templates completely, so that only particular parts can be modified.

**Comment by Jeffx9** [comment]
Hi Glutanimate,

Thank you for your prompt reply. As I was about to sleep, I did a quick search and found your trouble shooting guide. It occurred to me that uninstalling and reinstalling the programs and add ons would not actually fix the code. Therefore, I went back into my MacBook, opened anki and went to tools>cards to edit the script back to the “default” settings as recommended on your troubleshooting guide. This seemed to have fixed the issue.

I believe when I first tried to use it a few weeks ago, I must have stuffed the settings I by playing around with the script and it never reverted back to default, even after installing/reinstalling it.

Thank you so much for your help and effort, your program is going to help me so much in my medical studies in the future.

Kindest regards,

Jeff
